### PR TITLE
Update REDAME.md

### PR DESCRIPTION
**Pull request by AbhineshJha**
Fixed typo and grammar mistakes

### CODE REVIEW

**Comment by AbhineshJha** [comment]
please review @ofTheo

**Comment by dimitre** [comment]
in this PR there will be some broken SVG files like this one:
https://github.com/openframeworks/openFrameworks/workflows/build-macOS/badge.svg

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 openFrameworks is a C++ toolkit for creative coding.  If you are new to OF, welcome!
 
-- [Grab the Nightly Release (to avoid submodule cloning)](https://github.com/openframeworks/openFrameworks/releases)
+- [Grab the Nightly Releases (to avoid submodule cloning)](https://github.com/openframeworks/openFrameworks/releases)
```
Comment (ofTheo) [inline README.md:6]: release not releases

**Comment by ofTheo** [comment]
Still needs to go back to Release not Releases. 

Just curious do you use openFrameworks? 
Your commit history looks a lot like someone who is trying to build their profile with very small Pull Requests fixing questionable typos eg: `open source-project'

**Comment by AbhineshJha** [comment]
sorry for inconvenience , i am just a beginner in open source !!

**Pull request closed without merging**